src/OmniSharp.Host/Program.cs

```diff
@@ -73,6 +74,11 @@ public static void Main(string[] args)
                     enumerator.MoveNext();
                     serverInterface = (string)enumerator.Current;
                 }
+                else if(arg == "--encoding")
+                {
+                    enumerator.MoveNext();
+                    encoding=System.Text.Encoding.GetEncoding((string)enumerator.Current);
```

Comment:
Please add spaces around the `=` to match the formatting in the rest of the file.